index.js

```diff
@@ -54,7 +65,10 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  //could also loop through movies array summing all imdb ratings (which is what .reduce is doing here) into a new variable then divide by movies.length
+    return movies.length > 0 ? movies.reduce((total, movie) => total + Number(movie.imdbRating), 0) / movies.length : 0;
```

Comment:
This is a great reduce call but I would break these down to multiple lines on the `?` and possibly even the reduce call.